test/e2e/specs/site-editor/page-list.spec.js

```diff
@@ -53,4 +61,356 @@ test.describe( 'Page List', () => {
+			},
+			statusVisibility: {
+				performEdit: async ( page ) => {
+					const statusAndVisibility = page.getByLabel(
+						'Status & Visibility'
+					);
+					await statusAndVisibility.click();
+					const options = [
+						'Published',
+						'Draft',
+						'Pending Review',
+						'Private',
+					];
+
+					for ( const option of options ) {
+						await page
+							.getByRole( 'radio', { name: option } )
+							.click();
+						await expect( statusAndVisibility ).toContainText(
+							option
+						);
+
+						if ( option !== 'Private' ) {
+							await page
+								.getByRole( 'checkbox', {
+									name: 'Password protected',
+								} )
+								.check();
+						}
+					}
+				},
+				assertInitialState: async ( page ) => {
+					const statusAndVisibility = page.getByLabel(
+						'Status & Visibility'
+					);
+					await expect( statusAndVisibility ).toContainText(
+						'Published'
+					);
+				},
+				assertEditedState: async ( page ) => {
+					const statusAndVisibility = page.getByLabel(
+						'Status & Visibility'
+					);
+					await expect( statusAndVisibility ).toContainText(
+						'Private'
+					);
+				},
+			},
+			author: {
+				assertInitialState: async ( page ) => {
+					const author = page.getByLabel( 'Author' );
+					await expect( author ).toContainText( 'admin' );
+				},
+				performEdit: async ( page ) => {
+					const author = page.getByLabel( 'Author' );
+					await author.click();
+					const selectElement = page.locator(
+						'select:has(option[value="1"])'
+					);
+					await selectElement.selectOption( { value: '1' } );
+				},
+				assertEditedState: async () => {},
+			},
+			date: {
+				assertInitialState: async ( page ) => {
+					const dateEl = page.getByLabel( 'Edit Date' );
+					const date = new Date();
+					const yy = String( date.getFullYear() );
+
+					await expect( dateEl ).toContainText( yy );
+				},
+				performEdit: async ( page ) => {
+					const dateEl = page.getByLabel( 'Edit Date' );
+					await dateEl.click();
+					const date = new Date();
+					const yy = Number( date.getFullYear() );
+					const yyEl = page.locator(
+						`input[type="number"][value="${ yy }"]`
+					);
+
+					await yyEl.focus();
+					await page.keyboard.press( 'ArrowUp' );
+				},
+				assertEditedState: async ( page ) => {
+					const date = new Date();
+					const yy = Number( date.getFullYear() );
+					const dateEl = page.getByLabel( 'Edit Date' );
+					await expect( dateEl ).toContainText( String( yy + 1 ) );
+				},
+			},
+			slug: {
+				assertInitialState: async ( page ) => {
+					const slug = page.getByLabel( 'Edit Slug' );
+					await expect( slug ).toContainText( 'privacy-policy' );
+				},
+				performEdit: async ( page ) => {
+					const slug = page.getByLabel( 'Edit Slug' );
+					await slug.click();
+					await expect(
+						page.getByRole( 'link', {
+							name: 'http://localhost:8889/?',
+						} )
+					).toBeVisible();
+				},
+				assertEditedState: async () => {},
```

Comment:
Not for this PR, but we could try to add a test for when the slug is actually editable.